lib/membrane_element_rtp_aac/depayloader.ex

```diff
@@ -29,33 +33,22 @@ defmodule Membrane.RTP.AAC.Depayloader do
   end
 
   @impl true
-  def handle_stream_format(:input, _caps, _ctx, state) do
-    caps = %AAC{profile: state.profile, sample_rate: state.sample_rate, channels: state.channels}
-    {[stream_format: {:output, caps}], state}
-  end
-
-  @impl true
-  def handle_demand(:output, size, :buffers, _ctx, state) do
-    {[demand: {:input, size}], state}
+  def handle_stream_format(:input, _stream_fmt, _ctx, state) do
+    stream_fmt = %AAC{
```

Comment:
Do you think having `mode` in the `AAC` structure (from the membrane_aac_format repo) would be beneficial? I would say that we might think about it, as we could use this information, for instance, in the payloader (instead of passing the option).

Reply:
I did consider it but don't think this is the right approach considering it's an RTP specific parameter. If you think the improvement in conciseness outweighs the cons I can make the change